infrastructure/certificates/Pulumi.staging.yaml

```diff
@@ -1,5 +1,5 @@
+environment:
+  - certificates/staging
```

Comment:
so are we just setting this up on staging for now? or should we also edit the `Pulumi.production.yaml` files?